aten/src/ATen/native/GridSampler.h

```diff
@@ -38,16 +38,16 @@ static inline scalar_t grid_sampler_unnormalize(scalar_t coord, int64_t size,
 // except that it also returns the `d output / d input` via pointer argument
 // `grad_in`.
 // This is useful in the backward pass of grid_sampler.
-template <typename scalar_t>
-static inline scalar_t grid_sampler_unnormalize_set_grad(scalar_t coord, int64_t size,
-                                                         bool align_corners, scalar_t *grad_in) {
+template <typename coord_scalar_t, typename grad_scalar_t>
```

Comment:
Might consider renaming `grad_scalar_t` to `data_scalar_t` to make the changes consistent.